koswat/configuration/io/koswat_run_settings_importer.py

```diff
@@ -295,9 +298,16 @@ def _import_selected_dike_section_names(self, txt_file: Path) -> list[str]:
         if not txt_file.is_file():
             logging.error("Dike selection txt file not found at %s", txt_file)
             return []
+
         _reader = KoswatTxtReader()
         _reader.koswat_txt_fom_type = KoswatDikeSelectionTxtFom
-        return _reader.read(txt_file).dike_sections
+        _sections = _reader.read(txt_file).dike_sections
+
+        if not _sections:
+            logging.error("No dike sections found in %s", txt_file)
+            return []
```

Comment:
is this not the same? did you just extend this for the logging message? I would expect the error to be handled in the calling routine.